### DISCUSSION TITLE
New to Open Source

### DISCUSSION THREAD

**Discussion by nathannunes** [quarkusio/quarkus#35230] [Introductions]
Hi Everyone, 
I'm new to open source contributing, I've been using quarkus for a while now, but could someone guide me on something that I can actively contribute to here?

**Comment by geoand** [comment]
Hi,

Thanks for your interest in contributing!

The easiest way to start contributing is usually by finding issues in the [docs](https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc).
If one wants to move on to contributing code, the [good first issue](https://github.com/quarkusio/quarkus/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) label we have provides a list of issues we think make sense for folks news to the codebase.
If one does decide to contribute code, [this](https://github.com/quarkusio/quarkus/blob/main/CONTRIBUTING.md) provides some tips on how to work with it.

**Comment by MrRaveen** [comment]
Hello! also I have these links related to sources. check those also. 

https://github.com/quarkusio/quarkus/wiki

https://quarkus.io/guides/

https://quarkus.io/get-started/

https://github.com/quarkusio/quarkus/blob/574585e72eecd66fdb13e7b15b494462bf500e62/CONTRIBUTING.md#ide-config-and-code-style

https://github.com/quarkusio/quarkus/discussions

https://www.youtube.com/playlist?list=PLsM3ZE5tGAVbMz1LJqc8L5LpnfxPPKloO

https://stackoverflow.com/questions/tagged/quarkus

https://github.com/quarkusio/quarkus-insights

https://calendar.google.com/calendar/embed?src=935090aaae38be35eda437d018782b7827f3770f7a0344013677acd861570b20%40group.calendar.google.com&ctz=Europe%2FParis

https://www.youtube.com/playlist?list=PLsM3ZE5tGAVZ_rG48SFUFpEyTYGai0YLw

email contact :[quarkus-dev+subscribe@googlegroups.com](mailto:quarkus-dev+subscribe@googlegroups.com)

https://quarkusio.zulipchat.com/#recent

https://github.com/orgs/quarkusio/projects
